### ISSUE TITLE
Clean up Admin Landing Page

### ISSUE THREAD

**Issue by ClaireReinken** [FEISystems/ca-adpq-prototype#144]
Remove Test Links text

Delete all links except Inventory and Reports

Change Inventory to "Add, Edit, and Delete Items from the Catalog"

Add an image to the right side

Add text explaining functionality - Lisa will provide

**Comment by jennymnh** [comment]
I feel like changing "Inventory" into "Add, Edit, and Delete Items from the Catalog" would be too wordy. That could go into the text explaining what it is. I suggest simply leaving it as "Catalog"

Here is what I currently have it styled as. Let me know your comments/changes and I can check it in once we're done.
![image](https://cloud.githubusercontent.com/assets/9370407/23324061/a76722fa-fab9-11e6-9f6b-159f0ce94a76.png)

**Comment by ClaireReinken** [comment]
I like it! Please just use catalog

- Claire Reinken

On Feb 24, 2017, at 5:50 PM, Jenny Ho <notifications@github.com<mailto:notifications@github.com>> wrote:


I feel like changing "Inventory" into "Add, Edit, and Delete Items from the Catalog" would be too wordy. That could go into the text explaining what it is. I suggest simply leaving it as "Catalog"

Here is what I currently have it styled as. Let me know your comments/changes and I can check it in once we're done.
[image]<https://cloud.githubusercontent.com/assets/9370407/23324061/a76722fa-fab9-11e6-9f6b-159f0ce94a76.png>

-

**Comment by neelusingh28** [comment]
Verified and Passed in 5000 site.
